README.md

```diff
@@ -107,6 +107,22 @@ If not one of the well-described tests, the Fake Provider exhibits the following
 - If the CPR number ends in `000`, `001` or `002` it returns NotFound
 - The Fake provider returns random data, using the CPR number as the seed. This ensures recurring calls return the same response
 
+## CPR Subscription
+
+Whenever there is a change in information about a Citizen the CPR provider will raise events.
+ 
+Example -
+ [Datafordeler (Events)](https://confluence.datafordeler.dk/pages/viewpage.action?pageId=17137834#H%C3%A6ndelserp%C3%A5Datafordeleren-Brugsscenarier)
+ 
+- Subscribe To The Events By CPR number or by PersonId
+In Order to recieve events, you must subscribe with the `SubscriptionId` and `ConfigurationId` and the `CPR number` or by `CPR Person ID`  for which the events needs to be received.
+ 
+- Unsubscribe By CPR number or by PersonId
+To Stop receiveing the events, you must unsubscribe the previously created subscription either with `CPR number` or `CPR Person ID`.
+ 
+- Get Events
+To fetch the events for which you have subscribed, use Get Events along with your SubscriptionId and ConfigurationId and the desired time period.
```

Comment:
The unfiltered GetEvents method still exists doesn't it? If so, shouldn't we refer to something like GetFilteredEvents?

Reply:
Changed method name to GetFilteredEvents